tests/validators_/tests.py

```diff
@@ -0,0 +1,14 @@
+from django.core.exceptions import ValidationError
+from django.test import SimpleTestCase
+
+from django_mongodb_backend.fields.validators import LengthValidator
+
+
+class TestValidators(SimpleTestCase):
+    def test_validators(self):
+        validator = LengthValidator(10)
+        with self.assertRaises(ValidationError):
```

Comment:
I guess you may have copied the style from Django tests, but I would always include the message, not only to test the message, but to make the test more clear about which case(s) is/are tested.